docs/integrations/sigrid-api-documentation.md

```diff
@@ -211,7 +211,46 @@ Valid model names are:
 
 </details>
 
+### Security ratings
+
+You can obtain the security ratings, which are also available via Sigrid's portfolio and system dashboards,
+using the following end point:
+
+- `GET https://sigrid-says.com/rest/analysis-results/api/v1/model-ratings/{customer}?feature=SECURITY`
```

Comment:
It is actually somewhat more deluxe:
- We also have a system-level variant: `GET https://sigrid-says.com/rest/analysis-results/api/v1/model-ratings/{customer}/{system}?feature=SECURITY`.
- Both take an additional, optional query parameter: `endDate=YYYY-MM-DD`. This allows users to get the ratings on the given date.